### DISCUSSION TITLE
Does Jedis support discovery new nodes in Redis cluster?

### DISCUSSION THREAD

**Discussion by christranv** [xetorthio/jedis#3607] [Q&A]
I met an issue when a node in my Redis cluster is down and replaced by new nodes in K8s, Jedis can not detect new node as a replacement for failed one. Is there any way to resolve this problem?

**Comment by sazzad16** [comment]
It should be usually rediscovered.
In case #3596 resolves this issue, check latest 5.1.0-SNAPSHOT.
